openedx/features/partners/models.py

```diff
@@ -33,3 +33,11 @@ def __unicode__(self):
     class Meta:
         unique_together = ('user', 'partner')
 
+
+class PartnerCommunity(models.Model):
+    category_id = models.CharField(max_length=255)
+    partner = models.ForeignKey(Partner, db_index=True, on_delete=models.CASCADE, related_name="partner_communities")
```

Comment:
I think `related_name='communities'` make more sense, as this will always be accessed from **Partner** object. So no need to mention partner again.

Reply:
Done. Good suggestion